sub-packages/bionemo-esm2/src/bionemo/esm2/scripts/train_esm2.py

```diff
@@ -553,6 +570,13 @@ def get_parser():
         required=False,
         help="Path to nemo1 file, if desired to load at init time.",
     )
+    parser.add_argument(
+        "--disable-checkpointing",
+        action="store_false",
+        default=True,
+        dest="create_checkpoint_callback",
+        help="Disable creating a ModelCheckpoint callback.",
+    )
```

Comment:
Same applies:

Would you please mirror these changes in `sub-packages/bionemo-llm/src/bionemo/llm/train.py`